### ISSUE TITLE
can't delete node on PGC

### ISSUE THREAD

**Issue by mdorr75** [bpaauwe/udi-purpleair-poly#6]
Hello.  I can't delete a node on PGC.  I've deleted the key/value pair in the configuration menu, and I've checked the delete box on the node tab.  Both after stopping and then restarting the nodeserver.  Is there another step required?  Thanks

**Comment by bpaauwe** [comment]
The node server won't delete nodes.  However, deleting them via the Polyglot dashboard should work (you may have to also manually delete them from the ISY). 

If you haven't, try deleting them from the ISY before restarting the node server.  Polyglot may be re-adding them based on what's configured in the ISY.

If Polyglot isn't deleting them, then that would be a bug in Polyglot, not in the node server.

**Comment by mdorr75** [comment]
Indeed, I had not tried deleting from the ISY during my various attempts, but I ultimately resorted to a brute force re-install to resolve the issue.  Thanks for the response, though; I'll keep it in might in case of any future instances.
